Platforms/QemuQ35Pkg/.azurepipelines/NuGet.yml

```diff
@@ -0,0 +1,204 @@
+## @file
+# Azure Pipeline build file for publishing a NuGet package.
+#
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+
+parameters:
+- name: publish_q35_binaries
+  displayName: Publish QEMU Q35 Binaries
+  type: boolean
+  default: true
+- name: publish_q35sbsa_binaries
+  displayName: Publish QEMU Q35 SBSA Binaries
+  type: boolean
+  default: true
+
+name: $(Date:yyyyMMdd)$(Rev:.r) Mu Tiano Platforms NuGet Release
+
+jobs:
+  - job: Publish
+
+    strategy:
+      matrix:
+        ${{ if eq(parameters.publish_q35_binaries, true) }}:
+          QEMU Q35 Debug:
+            artifact_name: Binaries QemuQ35Pkg DEBUG
+            nuget_package_config_file: >-
+              https://raw.githubusercontent.com/microsoft/mu_tiano_platforms/main/Platforms/QemuQ35Pkg/NugetPublishing/DebugConfig.yaml

```

Comment:
Should this not reference the release tag instead of main?

Reply:
Given this is triggered on release publish, I assumed it would be the same. But it probably should be the tag. I'll need to double check that's available at the time the pipeline is run.

Reply:
after going through this more, it does seem we kind of need the assumption there are no commits between the release and this run, both here and for pulling artifacts, etc, so it may just make sense to be consistent. So feel free to ignore this comment.

Reply:
I did go ahead and update it to use the latest release tag.

Updated in PR commit https://github.com/microsoft/mu_tiano_platforms/pull/286/commits/90137807c540d298b67f3a543c3c2ebb47aa9f1a